Gemfile.lock

```diff
@@ -106,11 +106,11 @@ GEM
     bindex (0.8.1)
     binding_of_caller (1.0.0)
       debug_inspector (>= 0.0.1)
-    bootsnap (1.7.2)
+    bootsnap (1.7.3)
```

Comment:
### [bootsnap](https://github.com/Shopify/bootsnap)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.7.2 → 1.7.3

[_[change-log](https://github.com/Shopify/bootsnap/blob/master/CHANGELOG.md), [source-code](https://github.com/Shopify/bootsnap)_]





<details>
<summary>Commits</summary>

A change of **3** commits. See the full changes on [the compare page](https://github.com/Shopify/bootsnap/compare/v1.7.2...v1.7.3).

These are the individual commits:
- (aa9b800) [YAML cache: do not cache documents containing tags](https://github.com/Shopify/bootsnap/commit/aa9b800c6b27e889e6991c0c34c8c44558c70754)
- (91beaa6) [Merge pull request #351 from Shopify/psych-tags](https://github.com/Shopify/bootsnap/commit/91beaa6d66f45ba6b2aa957953476de7e438f65e)
- (a343dc2) [Release 1.7.3](https://github.com/Shopify/bootsnap/commit/a343dc2191d3b05fc83c01406ee11f2fa47bea91)


</details>